packages/shorebird_cli/lib/src/shorebird_flutter.dart

```diff
@@ -62,6 +64,30 @@ class ShorebirdFlutter {
     return status.isEmpty;
   }
 
+  /// Returns the current Shorebird Flutter version.
+  /// Throws a [ProcessException] if the version check fails.
+  /// Returns `null` if the version check succeeds but the version cannot be
+  /// parsed.
+  Future<String?> getVersion() async {
```

Comment:
Is there a file missing from this PR? This doesn't seem to be used.

Reply:
it's not used yet was going to refactor the code to use it in the next PR to keep the PRs smaller